scene/gui/tab_bar.cpp

```diff
@@ -994,6 +995,36 @@ int TabBar::get_tab_icon_max_width(int p_tab) const {
 	return tabs[p_tab].icon_max_width;
 }
 
+void TabBar::set_font_color_override_all(int p_tab, Color p_color) {
+	ERR_FAIL_INDEX(p_tab, tabs.size());
+
+	for (int i = 0; i < DrawMode::DRAW_MAX; i++) {
+		tabs.write[p_tab].font_color_overrides[i] = p_color;
```

Comment:
You should do `Tab &tab = tabs.write[p_tab]` before the loop. Using `.write` has a cost each time.